OasisDEX/automation-smartcontracts
# FEAT - Add get token task

`get-token` allow you to get 1000 of any token.
For now only RETH is on the list

How to test:
- run local hh node
- run `npx hardhat get-token --to your address --token RETH --network local` in another treminal
- check your balance
